HunterPie.Integrations/Datasources/MonsterHunterRise/Entity/Player/MHRWirebug.cs

```diff
@@ -137,6 +137,13 @@ private set
         remove => _onCooldownUpdate.Unhook(value);
     }
 
+    private readonly SmartEvent<MHRWirebug> _onMaxCooldownUpdate = new();
+    public event EventHandler<MHRWirebug> OnMaxCooldownUpdate
+    {
+        add => _onMaxCooldownUpdate.Hook(value);
+        remove => _onMaxCooldownUpdate.Unhook(value);
+    }
```

Comment:
Is there any technical reason for this to get it's own event? Like is it possible for the MaxCooldown to update without updating the current cooldown value?

Reply:
Yes, It is for fix temporary wirebug's cooldown border not showing on newly added.

Reply:
Hmm, I assume the reason for that is because the first time you get a wirebug, it's maximum cooldown is always at 0.0 and then it becomes 480.0 once you get the wirebug. I believe there's an easier way to fix that issue without relying on a new event in HunterPie.

I'll look into it

Reply:
I tried that, couldn't fix that problem. cause There is no triggered dispatch, when MaxCooldown value is changed.